### PR TITLE
Add Numerals Duo Neon Green watchface.

### PR DESCRIPTION
**Pull request by MagneFire**
This watchface was originally requested by @allendema never got around to actually pushing it to upstream ;)

~~edit: Fun fact, this uses `QtGraphicalEffects` for the `OpacityMask`s and it works fine with `tetra`, which had (hopefully no more) issues with `QtGraphicalEffects` :smile:~~ Turns out that after some revisions it now is broken for `tetra` (Only one digit is shown, the others are full color rectangles) :cry:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ The creators of the below listed watchfaces are happy to answer your questions a
 | ![thumbnail](.thumbnails/karlos-matrix-round.jpg) |![thumbnail](.thumbnails/karlos-matrix.jpg) | [karlos-matrix](karlos-matrix/usr/share/asteroid-launcher/watchfaces/karlos-matrix.qml) | [eLtMosen](https://github.com/eLtMosen) |
 | ![thumbnail](.thumbnails/logo-45degree-round.jpg) |![thumbnail](.thumbnails/logo-45degree.jpg) | [logo-45degree](logo-45degree/usr/share/asteroid-launcher/watchfaces/logo-45degree.qml) | [eLtMosen](https://github.com/eLtMosen) |
 | ![thumbnail](.thumbnails/motivational-quotes-round.jpg) |![thumbnail](.thumbnails/motivational-quotes.jpg) | [motivational-quotes](motivational-quotes/usr/share/asteroid-launcher/watchfaces/motivational-quotes.qml) | [eLtMosen](https://github.com/eLtMosen) |
+| ![thumbnail](.thumbnails/numerals-duo-neon-green-round.jpg) |![thumbnail](.thumbnails/numerals-duo-neon-green.jpg) | [orbiting-asteroids](numerals-duo-neon-green/usr/share/asteroid-launcher/watchfaces/numerals-duo-neon-green.qml) | [MagneFire](https://github.com/MagneFire) |
```
Comment (eLtMosen) [inline README.md:70]: Nice, that looks sleek af :D
Here is an [Orbiting Asteroids] string in the description that was likely not edited after copy+paste.

Comment (MagneFire) [reply README.md:70]: Thanks! Great catch too!

**Comment by eLtMosen** [comment]
neon4
That's what i mean.
VS all spacing on 19p
neon5
x

**Comment by MagneFire** [comment]
Had to do some messing around with the code to ensure that the padding is equal for all sides. But now it looks like this (Elongated to demonstrate that it can cope with different resolutions):
padding-fix

It will take some time for me to get the screenshots from the watches though.

**Comment by MagneFire** [comment]
Eh, where is that screenshot from?
The original images had this border which I removed (apparently failed).

**Comment by eLtMosen** [comment]
I cloned your MagneFire:neon repo to local and those images where in it during the time i did the clone.

Re: spacing, thats cool! Imho it might look even better with the spacing equalling the line thickness of the numerals.
Now they are quite far away from eachother and the combination 0 1 looks like the 1 falls off since it appears very far awa from the 0 due to it being right aligned in the png "box".
But that is minor and what i actually had a gripe with is fixed with equal spacing.
neon-few-spacing

**Comment by MagneFire** [comment]
Again, thanks for this feedback!
You made this watchface look better then I thought it would ever be!

This is what it now looks like:
equal-line-thickness

**Comment by eLtMosen** [comment]
Honestly, i saw the numerals, its simplicity and had the feeling of "stock material" right away :D
It is so much different to the current stock watchfaces but imo is spot on all asteroid design principles.
To go the extra mile one could look into how to make the letters in a really shiny subtile neon glow now that we have drop shadows. I think i might waste an evening on that :P

**Comment by eLtMosen** [comment]
For completeness so it does not get lost in the matrix flow, here is the neon effect i came up with:

neon4
[numerals-duo-neon-green.zip](https://github.com/AsteroidOS/unofficial-watchfaces/files/7471171/numerals-duo-neon-green.zip)
h

**Comment by MagneFire** [comment]
Oh wow! I only had a look at the actual code of that now.
Was sort of expecting it to be very complex, but it fits actually neatly into the current code already used in the PR!

Nonetheless your version looks quite different to the current one. Should we instead have a second version in the unofficial-watchfaces repo?

**Comment by eLtMosen** [comment]
That makes sense. I like both for what they are. Would be cool if you could open the PRs, i messed my Git folder and first want to get the things done i have in my pipe before starting new things :P

**Comment by FlorentRevest** [comment]
I'm glad we can use `QtGraphicalEffects` again. It makes for really cool watchfaces :) And tetra can go to hell :)

**Pull request merged**